c_sharp_json_diff/DoubleDiffProcessor.cs

```diff
@@ -0,0 +1,42 @@
+﻿using Newtonsoft.Json.Linq;
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace c_sharp_json_diff
+{
+    class DoubleDiffProcessor : DiffProcessor
+    {
+        public override Dictionary<string, object> PerformDiffProcess(JToken left, JToken right)
+        {
+            string leftValueAsString = left.ToString();
+            string rightValueAsString = right.ToString();
+            Dictionary<string, object> dict = new Dictionary<string, object>();
+
+            if (!IsIntable(leftValueAsString, rightValueAsString))
+            {
+                NumericDiff diff = new NumericDiff();
+
+                if (double.TryParse(leftValueAsString, out double leftValueAsDouble) && double.TryParse(rightValueAsString, out double rightValueAsDouble))
+                {
+                    diff.Subtraction = leftValueAsDouble - rightValueAsDouble;
```

Comment:
Change the order to right minus left and right / left. This is the usual way that deltas/diffs are calculated.